astro/src/content/quickstarts/quickstart-javascript-express-web.mdx

```diff
@@ -0,0 +1,259 @@
+
+## Authentication
+
+Make sure once again that you are in the directory `change-bank/src`.
+Create a file called `index.ts` and start by setting up your imports and constants. These constants have already been created by running the kickstart when running FusionAuth.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/index.ts"}
+  lang="ts"
+  tags="top"/>
+
+<Aside type="note">
+If you want to see where the FusionAuth values came from, they can be found in the 
+FusionAuth app (http://localhost:9011/admin). The tenant ID is found on the Tenants
+page. To see the Client Id and Client Secret, go to the Applications page and 
+click the `View` icon under the actions for the ChangeBank application. You'll 
+find the Client Id and Client Secret values in the `OAuth configuration` section.
+</Aside>
+
+### Static files
+
+You need a way to serve your static files, luckily Express makes this easy with `express.static`. These files will include all of the CSS and images that are required for the UI of the ChangeBank application. 
+
+Make sure that you are in the directory `change-bank` when running these commands to create new directories and copy the css and images required from `complete-application`.
+
+```shell
+mkdir static && mkdir static/css && mkdir static/img && 
+cp ../complete-application/static/css/changebank.css static/css/changebank.css && cp ../complete-application/static/img/money.jpg static/img/money.jpg
+```
+
+In your `index.ts` file include the below code to serve any assets in the static directory.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/index.ts"}
+  lang="ts"
+  tags="static"/>
+
+### Homepage
+
+In order to show off your homepage you need to create a route with the base URL `/`. In this route check to see if `userDetails` cookie has any user values and correctly respond with either the `home.html` or `account.html` template. Also if there is no user you can generate the PKCE challenge pair and store them into `userSession` cookie (this is an `HTTPOnly` cookie, inaccessible to JavaScript).
+
+<Aside type="note">
+In this file you will use the [pkce-challenge](https://www.npmjs.com/package/pkce-challenge) package to generate a Proof Key for Code Exchange (PKCE) challenge pair. Read more about [PKCE](/articles/oauth/modern-guide-to-oauth).
+</Aside>
+
+Include the below code into `index.ts`, so that the homepage can be displayed when a user is not logged in. If a user is logged in this route will redirect to the `/account` route.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/index.ts"}
+  lang="ts"
+  tags="homepage"/>
+
+### Login Route
+
+In the ChangeBank application there is a `Login` button which will open the `/login` route when clicked. This route redirect them to a login page managed by FusionAuth.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/src/index.ts"}
+  lang="ts"
+  tags="login"/>
+
+- `client_id`: Your `applicationId`
```

Comment:
You refer to it as a client Id above. This is the only time you refer to this as `applicationId`. I think we should refer to it as client Id.